src/app/child-dev-project/notes/attendance.model.ts

```diff
@@ -0,0 +1,21 @@
+
+export class AttendanceModel {
```

Comment:
This sounds very similar to our `AttendanceMonth` class and the (completely separate) system for keeping students' school attendance. Maybe `MeetingNoteAttendance` or something like that can make its purpose more clear?